BUILD

```diff
@@ -0,0 +1,364 @@
+load("//:build_tools/bazel/glob_lit_test.bzl", "glob_lit_tests")
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library", "td_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],
+)
+
+cc_library(
+    name = "base",
+    srcs = [
+        "dialect/Base.cpp",
+    ],
+    hdrs = [
+        "dialect/Base.h",
+    ],
+    #includes = ["."],
```

Comment:
Why is this line commented out? Same question for similar lines below.

Reply:
I did this to avoid the error 
```
BUILD:9:11: in includes attribute of cc_library rule //:base: '.' resolves to the workspace root, which would allow this rule and all of its transitive dependents to include any file in your worksp│remote: Resolving deltas: 100% (1/1), completed with 1 local object.
ace. Please include only what you need
```

If this looks good I will remove the commented lines.

Reply:
Looks good as a first approximation. I don't think this will work when StableHLO is built as part of another project, e.g. MLIR-HLO (definitely doesn't work in Blaze, didn't try with Bazel), but we'll figure this out during integrates.

Reply:
https://github.com/openxla/stablehlo/pull/48 should resolve this issue when it lands.